inc/utils/content.php

```diff
@@ -336,6 +336,23 @@ function echo_post_meta($the_post, $show_elements = array('date','sources','cate
   					}
   			}
       endif; ?>
+
+			<?php if (in_array('author', $show_elements) && get_post_type() !='news-article'):
```

Comment:
@S-mardii why are you checking here that the post is not of type **news-article**? I would make this logic applicable to all post types (not only to announcements). Please delete ```get_post_type() !='news-article')``` if there is no major reason to keep it.